whosonfirst/Dockerfile

```diff
@@ -3,16 +3,19 @@ FROM pelias/baseimage
 
 # downloader apt dependencies
 # note: this is done in one command in order to keep down the size of intermediate containers
-RUN apt-get update && apt-get install -y bzip2 && rm -rf /var/lib/apt/lists/*
+RUN apt-get update && apt-get install -y bzip2 && apt-get install -y unzip && rm -rf /var/lib/apt/lists/*
 
 # clone repo
 RUN git clone https://github.com/pelias/whosonfirst.git /code/pelias/whosonfirst
 
 # change working dir
 WORKDIR /code/pelias/whosonfirst
 
+# consume the build variables
+ARG REVISION=production
+
+# switch to desired revision
+RUN git checkout $REVISION
+
 # install npm dependencies
 RUN npm install
-
-# run tests
-RUN npm test
```

Comment:
running the tests should be fine here, I like the idea that images fail to create if the tests fail